code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -8,7 +8,8 @@
 	var/t_is = p_are()
 
 	var/msg = "<span class='info'>*---------*\nThis is <EM>[src.name]</EM>!\n"
-
+	if(isdwarf(src))

```

Comment:
There's no doubt more modular/efficient ways to achieve this effect.

Do examines happen enough for that to matter, though?